cmds/core/cpio/cpio.go

```diff
@@ -40,46 +41,45 @@ var (
 	debug  = func(string, ...interface{}) {}
 	d      = flag.Bool("v", false, "Debug prints")
 	format = flag.String("H", "newc", "format")
+
+	errInvalidArgs = errors.New("Usage of the command:\ncpio o < name-list [> archive]\ncpio i [< archive]\ncpio p destination-directory < name-list\nOptions: -H format (default: newc) -v Debug prints ")
 )
 
-func usage() {
-	log.Fatalf("Usage: cpio")
+func usage() error {
+	return errInvalidArgs
 }
 
-func main() {
-	flag.Parse()
-	if *d {
+func run(args []string, stdin *os.File, stdout io.Writer, d bool, format string) error {
+	if d {
 		debug = log.Printf
 	}
 
-	a := flag.Args()
-	debug("Args %v", a)
-	if len(a) < 1 {
-		usage()
+	debug("Args %v", args)
+	if len(args) < 1 {
+		return usage()
 	}
-	op := a[0]
+	op := args[0]
 
-	archiver, err := cpio.Format(*format)
+	archiver, err := cpio.Format(format)
 	if err != nil {
-		log.Fatalf("Format %q not supported: %v", *format, err)
+		return fmt.Errorf("Format %q not supported: %v", format, err)
```

Comment:
be sure to use %w to wrap the error(s)

Reply:
I fixed the wrapping in the newest commit.